src/gsi/crtm_interface.f90

```diff
@@ -667,6 +667,21 @@ subroutine init_crtm(init_pass,mype_diaghdr,mype,nchanl,nreal,isis,obstype,radmo
    error_status = crtm_channelinfo_subset(channelinfo(1), &
         channel_subset = nuchan(subset_start:subset_end))
 
+!JAJ the correct one    else if (channelinfo(1)%sensor_id(1:7) == 'iasi-ng' .AND. isis(1:7) == 'iasi-ng') then
```

Comment:
The CRTM coefficient files have the instrument name in the beginning of the file.  This is added to the final file.  This is extremely annoying.  This is also in the read_iasing.f90 file.  Unfortunately, I found errors in the spectral coefficient file so it will be a "while" before it is resolved.  Yes, this (and the one in read_iasing.f90) are TODO.  I will add this to the file(s)